packages/blade/src/components/Dropdown/DropdownOverlay.native.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { componentIds } from './dropdownUtils';
+import { useDropdown } from './useDropdown';
+import Box from '~components/Box';
+import { makeSize } from '~utils';
+
+const StyledDropdownOverlay = styled(Box)((props) => ({
+  transform: `translateY(${makeSize(props.theme.spacing[3])})`,
```

Comment:
does this works on ios and android both since its static?

Reply:
yup seems to work